package.json

```diff
@@ -91,14 +93,20 @@
     "url": "git@github.com:documentationjs/documentation.git"
   },
   "scripts": {
+    "build": "rm -rf lib && babel -D src -d lib",
     "release": "standard-version",
     "precommit": "lint-staged --verbose",
-    "format": "prettier --write '{lib,test,declarations,bin,default_theme}/**/*.js' --single-quote",
+    "format": "prettier --write '{src,__tests__,declarations,bin,default_theme}/**/*.js' --single-quote",
     "doc": "./bin/documentation.js build lib/index.js -f md --access=public > docs/NODE_API.md",
     "changelog": "standard-changelog -i CHANGELOG.md -w",
     "self-lint": "node ./bin/documentation.js lint",
-    "test": "eslint lib && are-we-flow-yet lib && flow check && npm run self-lint && npm run test-tap",
-    "test-tap": "tap -t 120 --coverage --nyc-arg=--cache test/*.js test/lib test/streams"
+    "test": "eslint src && are-we-flow-yet src && flow check && npm run self-lint && jest --runInBand"
```

Comment:
`--runInBand` makes tests way more reliable and faster on CircleCI, but slower locally. I'd love to find a way to only enable it in the CircleCI environment, but haven't found one yet.